app/views/users/show.html.erb

```diff
@@ -0,0 +1,104 @@
+<% if current_user.status %>
+<div class="px-6 y py-2">
+  <div class="text-justify py-4 mt-2.5">
+    <%= render "shared/header_stamp" %>
+    <h3>Mi cuenta</h3>
+  </div>
+
+  <div class="grid lg:grid-cols-2 md:grid-cols-1 mt-2.5">
+    <div clas="lg:col-start-1 lg:col-end-2">
+      <h5>Nombre completo</h5>
+      <p><%= current_user.name %></p>
+    </div>
+
+    <div clas="col-start-2 col-end-3">
+      <h5>Correo electrónico</h5>
+      <p><%= current_user.email %></p>
+    </div>
+  </div>
+
+  <div class="text-justify py-4 mt-2.5">
+    <h3>Mis negocios</h3>
+  </div>
+  
+  <hr class="w-full border-scene-neutral">
+
+       
+  <% MarketPlace.where(user_id: current_user.id).each do |market| %>
```

Comment:
WDYT about moving this to the controller?
Let's try to use views only for presenting data and let's try to avoid having logic or DB related stuff from here 😄